resources/integration_tests/kms_keyting.tf

```diff
@@ -0,0 +1,29 @@
+resource "google_kms_key_ring" "example-keyring" {
```

Comment:
nit: file name should be kms_keyRing.tf probably :)

Reply:
the tf files in integration_tests are just my test tf files that I used for resources implementation.